Source/SIMPLib/Geometry/GeometryHelpers.h

```diff
@@ -54,6 +54,22 @@
 namespace GeometryHelpers
 {
 
+/**
+ * @brief The LengthUnit class
+ */
+class LengthUnit
```

Comment:
Do we want to make this a namespace instead?  No object of this type can exist as there is no access to the constructor, and there are only static methods available.

Reply:
Made the changes along with a slightly different design that works better.